core/services/ardupilot_manager/main.py

```diff
@@ -1,6 +1,34 @@
 #! /usr/bin/env python3
+from typing import Any, Dict, List
+
+import uvicorn
 from ArduPilotManager import ArduPilotManager
+from fastapi import FastAPI
+from mavlink_proxy.Endpoint import Endpoint, EndpointType
+
+app = FastAPI()
+autopilot = ArduPilotManager()
+
+
+@app.get("/available_endpoints", response_model=List[Dict[str, Any]])
+def get_available_endpoints() -> Any:
+    return [endpoint.__dict__ for endpoint in autopilot.get_endpoints()]
```

Comment:
Can return a pretty json over a plain text ?

Reply:
It is already. FastAPI parse dicts into JSONs internally.

Reply:
I'm talking about pretty json over plain json, as pretty I'm saying: `JSON.stringify(json, null 2)`